docs/plugins/analysis-phonetic.asciidoc

```diff
@@ -65,6 +65,14 @@ GET phonetic_sample/_analyze
 
 <1> Returns: `J`, `joe`, `BLKS`, `bloggs`
 
+It is important to note that `"replace": false` can lead to unexpected behavior since
+the original and the phonetically analyzed version are both kept at the same token position.
+Some queries handle these stacked tokens in special ways. For example, the fuzzy `match`
+query does not apply <<fuzziness>> to stacked synonym tokens. This can lead to issues that are
```

Comment:
I tried to build these plugin docs, but I get the error " IDREF attribute linkend references an unknown ID "fuzziness"".

Reply:
Yep, you are right, doesn't work here as well. I changed it to using "{ref}" which is what I think we use in cases where we link between two "books" in the docs.